### DISCUSSION TITLE
GitHub student account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#67753] [GitHub Education]
### Hi everyone,

Good afternoon, I was wondering when I will be approved for my github student developer pack,  it's saying it should be approved soon but it's been about two days. So I am wondering the waiting period. 

<img width="1512" alt="Screenshot 2023-09-20 at 4 58 02 PM" src="https://github.com/community/community/assets/145393594/f9bca61d-9c7e-48d1-ae27-8125c49005c5">

**Comment by XiaoCdeGitHub** [comment]
I'm having the same problem, but the work order request remains unanswered!

**Reply by ghost** [reply]
Yea, i'm not sure what's taking so long wonder if there's a support team that handles this or not

**Comment by mantasu** [comment]
Hey, I had the same issue, you may still have to wait for a few more days as there will be some time before your benefits become active. When "Approved" tag becomes purple, you should be ready:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

If you don't receive any benefits after 2 weeks, just open a [ticket](https://support.github.com/request/education?tags=docs-generic).
